packages/dx-react-grid/docs/reference/table-tree-column.md

```diff
@@ -104,7 +104,7 @@ TableTreeColumn.Indent | [TableTreeColumn.IndentProps](#tabletreecolumnindentpro
 TableTreeColumn.ExpandButton | [TableTreeColumn.ExpandButtonProps](#tabletreecolumnexpandbuttonprops) | A component that renders a button used to controls a row's expanded state.
 TableTreeColumn.Checkbox | [TableTreeColumn.CheckboxProps](#tabletreecolumncheckboxprops) | A component that renders a checkbox used to control selection.
 
-The additional properties are added to the component's root element.
```

Comment:
Specifies whether to render selection controls. Requires the [SelectionState] dependency.
Specifies whether to render Select All checkbox. Requires the [IntegratedSelection] dependency.

Something cannot require a dependency. A dependency means it is a requirement. Rewrite.

Reply:
Requires the [...] plugin.